test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -1481,9 +1487,6 @@ TEST_F(ClientLbInterceptTrailingMetadataTest, InterceptsRetriesEnabled) {
 }  // namespace grpc
 
 int main(int argc, char** argv) {
-  // Make the backup poller poll very frequently in order to pick up
-  // updates from all the subchannels's FDs.
-  GPR_GLOBAL_CONFIG_SET(grpc_client_channel_backup_poll_interval_ms, 1);
```

Comment:
Why did this one need to move to SetUpTestCase?

Reply:
(similar question for other uses)

Reply:
[GTMGoogleTestRunner](https://github.com/google/google-toolbox-for-mac/blob/master/UnitTesting/GTMGoogleTestRunner.mm) is used to convert googletest cases to XCTest that can be run on iOS. GTMGoogleTestRunner doesn't execute the main() function, so any setup/teardown in main needs to be moved to SetUpTestCase/TearDownTestCase.

Reply:
GTMGoogleTestRunner calls [InitGoogleTest](https://github.com/google/google-toolbox-for-mac/blob/master/UnitTesting/GTMGoogleTestRunner.mm#L151), so we can leave it in main. 
grpc::testing::TestEnvironment just does some test initialization (install crash handler, seed RNG) that's not strictly required to run testcases on iOS, so I left it in main.

Reply:
Can you document this somewhere? I guess it will be self-documenting when the tests start failing but otherwise might be hard to pick up on. Like maybe adding a README.md in test/cpp so that anyone that wants to do tests later knows what they need to do differently compared to the typical googletest route? I'm not going to hold up this PR for that reason, but please do this in a followon PR.

Reply:
So things that are worth mentioning is that any test that wants to do any setup will have to switch from TEST to TEST_F as a result, that ASSERT_DEATH has to be changed, etc.

Reply:
Thanks @vjpai! I'll add a README in a followon PR.